conftest.py

```diff
@@ -220,14 +215,16 @@ def _load_module(module_name, path):
 
 @pytest.fixture(autouse=True)
 def reset_config():
-    """Reset the configuration directory used in the tests (tests.constants.ROOT_FS) after every test
+    """Reset the configuration directory used in the tests (tests.constants.ROOT_FS) before every test
     """
     # Ideally this would just be constants.ROOT_FS.removetree("/"), but SQLAlchemy complains if the SQLite file is deleted.
+    # We have between the workers a shared constants.ROOT_DIR
     for e in constants.ROOT_FS.walk.files(exclude=["mscolab.db"]):
         constants.ROOT_FS.remove(e)
+    # mscolab_settings, mswms_settings, testdata is not changed so we can keep it
     for e in constants.ROOT_FS.walk.dirs(search="depth"):
-        constants.ROOT_FS.removedir(e)
-
+        if not e in ("/mscolab", '/mswms', '/mswms/testdata'):
+            constants.ROOT_FS.removedir(e)
```

Comment:
Apart from my concerns that the assumption that these paths are not modified doesn't hold true, I would also suggest splitting these paths out of the configuration directory (and by extensions this fixture). Test data is not part of the configuration, and if it really is static it could just be a separate session-scoped fixture. I am not sure what is stored in `/mscolab` and `/mswms` and if it qualifies as "configuration".

Reply:
Also, as it is this change doesn't seem to speed up the tests, so I don't see what the motivation for it might be.

Reply:
This fixture seems to run in front of the test now. It removes the created server settings and seems not to be able to reload the new existing ones.

I can't currently see why that behaviour is such different.

Reply:
The behaviour is different because you have moved `MSWMS_SERVER_CONFIG_FS` and `DATA_FS` from msui/ to mswms/. `generate_initial_config` creates msui/ at the beginning, but doesn't create mswms/. The missing mswms/ directory is the issue.